src/main/java/frc/robot/subsystems/ShooterSubsystem.java

```diff
@@ -64,6 +67,9 @@ public ShooterSubsystem() {
         new MotorController("Flywheel", Constants.Shooter.shooterID, Constants.Shooter.kPIDFArray);
     flywheel2Controller = new MotorController("Flywheel 2", Constants.Shooter.shooter2ID);
     flywheelPID = flywheelController.getPIDCtrl();
+    flywheelFF =
+        new SimpleMotorFeedforward(
+            Constants.Shooter.kSg, Constants.Shooter.kVg, Constants.Shooter.kAg);
```

Comment:
This is confusing as flywheelFF isn't actually used.

Reply:
It's something I was experimenting with when I realized I broke main on accident
Will not have an effect